src/views/GlossaryPage.vue

```diff
@@ -0,0 +1,279 @@
+<template>
+  <DashboardLayout>
+    <DashboardCard>
+      <template #title> Agencies and Organizations </template>
+      <template #content>
+        <div class="px-4" style="font-size: 0.95rem">
+          <p v-for="agency in agencyAcronyms" :key="agency.acronym">
+            <strong style="font-size: 1.02rem">{{ agency.acronym }}:</strong>
+            {{ agency.name }}
+          </p>
+        </div>
+      </template>
+    </DashboardCard>
+    <DashboardCard>
+      <template #title>Models and Data</template>
```

Comment:
this title is kind of vague - I wonder if it would be better to make it an `Other` and move it to the last card?